rftrace/src/backend.rs

```diff
@@ -28,19 +30,28 @@ static OVERWRITING: AtomicBool = AtomicBool::new(false); // should the ring-buff
 static INDEX: AtomicUsize = AtomicUsize::new(0);
 static mut EVENTS: Option<&mut [Event]> = None;
 
-// !! Will always be initialized to all 0 by the OS, no matter what. This is just to make the compiler happy
-#[thread_local]
```

Comment:
What prevents the usage of https://doc.rust-lang.org/std/macro.thread_local.html here?